CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+<!--
+Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Changelog
+
+## 23.06 (2023-06-29)
```

Comment:
We need to be using MA version, not DLFW release number (although we could also include DLFW information)

Reply:
Understood and fixed.